compiler/util/misc.cpp

```diff
@@ -197,8 +205,52 @@ printDevelErrorHeader(BaseAST* ast) {
   }
 
 
-  if (ast && ast->linenum())
-    fprintf(stderr, "%s:%d: ", cleanFilename(ast), ast->linenum());
+  if( ast && ast->linenum() ) {
+    have_ast_line = 1;
```

Comment:
I request that a space be added after 'if' - above and in two more if-statements that you added below.